lib/BlockedBy.js

```diff
@@ -154,6 +173,13 @@ class BlockedBy extends NodeResque.Plugin {
         // enqueueIn uses milliseconds
         await this.queueObject.enqueueIn(
             this.reenqueueWaitTime() * 1000 * 60, this.queue, this.func, this.args);
+
+        await helper.updateBuildStatus({
```

Comment:
What if the job got reEnqueue multiple times? This call will fail cuz you cannot change the build status multiple times

Reply:
We need to make changes to API to allow it then